src/opam_docker.ml

```diff
@@ -94,7 +94,13 @@ module Cmds = struct
     from ~tag:distro "ocaml/opam2" @@
     add_cache_dir @@
     run "opam switch %s" (Oversions.to_string ocaml_version) @@
-    run "opam install -yv opam-depext"
+    run "opam install -yv opam-depext%s"
+      (if Oversions.older_than_4_06 ocaml_version
+       then " ocamlfind-secondary" (* NOTE: This is needed since dune 2.0.0
```

Comment:
This would be better to be `ocaml-secondary-compiler` which is the slow one - otherwise you’re always pulling in `ocamlfind` too

Reply:
oh, right, indeed.